x/gov/types/deposit.go

```diff
@@ -50,5 +43,5 @@ func (d Deposits) String() string {
 
 // Empty returns whether a deposit is empty.
 func (d Deposit) Empty() bool {
-	return d.String() == Deposit{}.String()
+	return NewCoins(d.Amount).Empty()
```

Comment:
Should we also check depositor and proposalID?

Reply:
Potentially. It depends on what this method is used for

Reply:
And the answer looks to be nowhere

Reply:
ahah. I'm fine to remove it completely even, in this case